src/components/App/App.js

```diff
@@ -12,26 +12,38 @@ import NoResults from '../NoResults/NoResults';
 import NewsCardList from '../NewsCardList/NewCardList';
 import SavedNews from '../SavedNews/SavedNews';
 import Popup from '../Popup/Popup';
+import ProtectedRoute from '../ProtectedRoute/ProtectedRoute';
 import RegistrationSuccess from '../RegistrationSuccess/RegistrationSuccess';
 import Footer from '../Footer/Footer';
 import SignUpForm from '../SignUpForm/SignUpForm';
 import SignInForm from '../SignInForm/SignInForm';
 
+import { CurrentUserContext } from '../../contexts/CurrentUserContext';
 import { searchForNews } from '../../utils/NewsApi';
+import * as mainApi from '../../utils/MainApi';
 
 function App() {
   const [currentPage, setCurrentPage] = React.useState(
     localStorage.getItem('')
```

Comment:
`    localStorage.getItem('')
`

I think this has no effect


you don't have anything in `localStorage` that can be taken with `''`


maybe you wanted like that


```js


  localStorage.getItem('currentPage')
```